packages/graph-engine/src/nodes/series/alternating.ts

```diff
@@ -49,21 +49,6 @@ export default class NodeDefinition extends Node {
 		this.addOutput('array', {
 			type: arrayOf(NumberSchema)
 		});
-		this.addOutput('indexed', {
-			type: {
-				$id: `https://schemas.tokens.studio/studio.tokens.series.alternating/indexed.json`,
-				type: 'object',
-				properties: {
-					index: {
-						type: NumberSchema
-					},
-					value: {
-						type: NumberSchema
-					}
-				}
-			},
-			visible: false
-		});
 	}
 
 	execute(): void | Promise<void> {
```

Comment:
Rather do 
```
const values = sequence.map((num, i) => {
	const patternIndex = i % patternToUse.length;
	const patternValue = patternToUse[patternIndex];
	return setToPrecision(num * patternValue, precision);
});
```
and then `this.outputs.array.set(values);`

Reply:
Similarly for the others, theres no need to push to an index array and then map across it again to pull the values out